R/synthetic_news_data.R

```diff
@@ -0,0 +1,38 @@
+#' Synthetic National Early Warning Scores Data
+#'
+#' Synthetic NEWS data(\href{https://www.rcplondon.ac.uk/projects/outputs/national-early-warning-score-news-2}) as the results of the NHSR_synpop package.
```

Comment:
\href{} is not necessary as vignettes are rendered with markdown - this will remove formatting warning in Rcmd check